README.adoc

```diff
@@ -3,164 +3,159 @@
 
-== Enable Scheduling
+== Running the Application
 
-Although scheduled tasks can be embedded in web applications and WAR files, the simpler
-approach (shown in the next listing) creates a standalone application. To do so,
-package everything in a single, executable JAR file, driven by a good old Java `main()`
-method. The following listing (from
-`src/main/java/com/example/schedulingtasks/SchedulingTasksApplication.java`) shows the
-application class:
+You should now be able to run the application by executing the main method in `SchedulingTasksApplication`. You can run the program from your IDE, or by executing the following Gradle command in the project root directory:
 
 ====
-[source,java]
 ----
-include::complete/src/main/java/com/example/schedulingtasks/SchedulingTasksApplication.java[]
+./gradlew bootRun
 ----
 ====
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/spring-boot-application-new-path.adoc[]
+This will start the application and the method annotated with `@Scheduled` will execute. You will see log messages similar to:
 
-The
-https://docs.spring.io/spring/docs/current/spring-framework-reference/htmlsingle/#scheduling-enable-annotation-support[`@EnableScheduling`]
-annotation ensures that a background task executor is created. Without it, nothing gets
-scheduled.
+====
+----
+20yy-mm-ddT07:23:01.665-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:01
+20yy-mm-ddT07:23:06.663-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:06
+20yy-mm-ddT07:23:11.663-04:00  INFO 19633 --- [   scheduling-1] c.e.schedulingtasks.ScheduledTasks       : The time is now 07:23:11
+----
+====
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_subhead.adoc[]
+NOTE: This example uses https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/annotation/Scheduled.html#fixedRate()[`fixedRate()`^] scheduling, so the application will run indefinitely until you interrupt the execution manually.
 
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/main/build_an_executable_jar_with_both.adoc[]
+== Testing with the `awaitility` Dependency
```

Comment:
Check with Damien whether code in headings works right. Ever since I started, we haven't been able to use it, because it doesn't render correctly.